Microsoft.Toolkit.Uwp.SampleApp/SamplePages/WeatherLiveTileAndToast/WeatherLiveTileAndToastPage.xaml

```diff
@@ -35,35 +44,80 @@
             </RichTextBlock>
 
             <!--  Toast Preview  -->
-            <visualizer:PreviewToast x:Name="PreviewToastWeather"
+            <StackPanel
```

Comment:
We could use a ContentPresenter and a template here to help facilitate the same look and feel across these samples using this layout?